parlai/core/params.py

```diff
@@ -458,6 +458,13 @@ def add_websockets_args(self):
         websockets.add_argument(
             '--port', default=35496, type=int, help='Port to run the websocket handler'
         )
+        websockets.add_argument(
+            '--password',
```

Comment:
i'm not sure if it's there yet, but this should be eventually wrapped up in some `add_chat_service_args` function

Reply:
Created the function and put common args between the services in it